client/src/app/components/Navbar/index.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+
+import AppBar from '@material-ui/core/AppBar';
+import IconButton from '@material-ui/core/IconButton';
+import makeStyles from '@material-ui/core/styles/makeStyles';
+import Toolbar from '@material-ui/core/Toolbar';
+import Typography from '@material-ui/core/Typography';
+import MenuIcon from '@material-ui/icons/Menu';
+
+const useStyles = makeStyles(theme => ({
+    menuButton: {
+        marginRight: theme.spacing(2),
+    },
+    title: {
+        flexGrow: 1,
+    },
+}));
+
+interface Props {
+    handleDrawerOpen: () => void;
+}
+
+const Navbar: React.FunctionComponent<Props> = ({ handleDrawerOpen }) => {
```

Comment:
:student: Teach me how to read this

Reply:
`Navbar` is of type `React.FunctionComponent<Props>` which is a generic type :gasp:, taking a type arg of `Props`. In this case, the type argument defines the shape of the props that get passed into the component.